datadog/src/main/scala/zio/metrics/connectors/datadog/DatadogEncoder.scala

```diff
@@ -27,7 +37,11 @@ case object DatadogEncoder {
     (key, values) => Chunk.fromArray(encodeHistogramValues(key, values).toString().getBytes())
   }
 
+  @deprecated("Use the overload that accepts DatadogPublisherConfig instead", "2.4.0")
   private def makeStatsdEncoder(config: DatadogConfig): StatsdEncoder =
+    makeStatsdEncoder(DatadogConfig.toPublisherConfig(config))
```

Comment:
This is a private method. We don't need to worry about binary compatibility in private methods so you can simply delete it